apiary.apib

```diff
@@ -7487,13 +7487,14 @@ Creates a new configuration row from an existing row version in [Development Bra
 
 ## Create or List Configuration Workspaces [/v2/storage/components/{component_id}/configs/{config_id}/workspaces]
 
-### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces]
+### Create Configuration Workspaces [POST /v2/storage/branch/{branch_id}/components/{component_id}/configs/{config_id}/workspaces?async={async}]
```

Comment:
chtelo by to poznamku o tom, ze async=false je depreacted

Reply:
Idealně u obou (i u standardní ws create), pokud není...

Reply:
Pridal som to a este som si vsimol, ze som nemal async=true response u toho configuration endpointu tak som to tam pridal. Snad to bude takto cajk.